### PR TITLE
[elasticsearch] Fix collection of shard recovery total time

### PR DESCRIPTION
**Pull request by miltonhultgren**
NOTE: This cannot be tested until the [Metricbeat changes](https://github.com/elastic/beats/pull/34653) are merged.

Related issue: https://github.com/elastic/beats/issues/34427

Since Metricbeat now reports the total time for a recovery, I've added a missing alias.

Related PRs:
https://github.com/elastic/elasticsearch/pull/94062
https://github.com/elastic/kibana/pull/151983

### CODE REVIEW

File: packages/elasticsearch/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.2.1"
+  changes:
+    - description: Fix collection of shard recovery total time
+      type: bugfix
+      link: TBD
```
Comment (crespocarlos) [inline packages/elasticsearch/changelog.yml:6]: Just a reminder to update this: https://github.com/elastic/integrations/pull/5367

File: packages/elasticsearch/manifest.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: elasticsearch
 title: Elasticsearch
-version: 1.4.0
+version: 1.4.1
```
Comment (miltonhultgren) [inline packages/elasticsearch/manifest.yml:3]: @klacabane  Or should I give it some kind of `preview` tag?

File: packages/elasticsearch/manifest.yml [hunk 3]
```diff
@@ -12,7 +12,7 @@ format_version: 1.0.0
 license: basic
 categories: ["elastic_stack", "datastore"]
 conditions:
-  kibana.version: ^8.7.0
+  kibana.version: ^8.8.0
```
Comment (miltonhultgren) [inline packages/elasticsearch/manifest.yml:15]: I'm actually bumping this to align with having Metricbeat 8.8 available which has the data that fills these aliases but I guess there isn't a way to express that and we should assume the stack is updated as a whole?

File: packages/elasticsearch/manifest.yml (hunk 2 above)
Comment (klacabane) [reply packages/elasticsearch/manifest.yml:3]: bumping patch makes sense to me. The change is trivial and is safe to release right away

Comment (miltonhultgren) [reply packages/elasticsearch/manifest.yml:3]: Okay, if you can hit approve I can merge 👍🏼

File: packages/elasticsearch/manifest.yml (hunk 3 above)
Comment (klacabane) [reply packages/elasticsearch/manifest.yml:15]: Not sure how deep this condition is enforced, but I'm wondering if we should bump the required version here. Is there any downside to publishing this alias for versions that don't have the source property populated ?

**Review by klacabane — APPROVED** [review]

**Pull request merged**